2017-11-22T08:06:40.925Z ERROR   HLFConnectionManager    :fabric-client()          [client-utils.js]: sendPeersProposal - Promise is rejected: Error: chaincode error (status: 500, message: Error: Failed to add object with ID 'Ankur' as the object already exists)
    at /usr/local/lib/node_modules/composer-rest-server/node_modules/grpc/src/node/src/client.js:554:15 {}$
2017-11-22T08:06:40.926Z ERROR   HLFConnection           :invokeChainCode()        {"message":"Error trying invoke business network. Error: chaincode error (status: 500, message: Error: Failed to add object with ID 'Ankur' as the object already exists)","stack":"Error: Error trying invoke business network. Error: chaincode error (status: 500, message: Error: Failed to add object with ID 'Ankur' as the object already exists)\n    at _initializeChannel.then.then.then.then.catch (/usr/local/lib/node_modules/composer-rest-server/node_modules/composer-connector-hlfv1/lib/hlfconnection.js:855:34)\n    at <anonymous>"}$
2017-11-22T08:09:42.042Z ERROR   HLFConnectionManager    :fabric-client()          [client-utils.js]: sendPeersProposal - Promise is rejected: Error: chaincode error (status: 500, message: Error: Failed to add object with ID 'Nehal' as the object already exists)
    at /usr/local/lib/node_modules/composer-rest-server/node_modules/grpc/src/node/src/client.js:554:15 {}$
2017-11-22T08:09:42.043Z ERROR   HLFConnection           :invokeChainCode()        {"message":"Error trying invoke business network. Error: chaincode error (status: 500, message: Error: Failed to add object with ID 'Nehal' as the object already exists)","stack":"Error: Error trying invoke business network. Error: chaincode error (status: 500, message: Error: Failed to add object with ID 'Nehal' as the object already exists)\n    at _initializeChannel.then.then.then.then.catch (/usr/local/lib/node_modules/composer-rest-server/node_modules/composer-connector-hlfv1/lib/hlfconnection.js:855:34)\n    at <anonymous>"}$
